### PR TITLE
[sailfish-browser] Use bigger tile size for high dpi devices. Fixes JB#29698

### PR DESCRIPTION
**Pull request by rainemak**
Review : @mikkoharju @rojkov @tworaz

### CODE REVIEW

**Comment by tworaz** [comment]
LGTM

**Comment by rojkov** [comment]
With this patch applied the system easily becomes unresponsive when https://medium.com/message/everything-is-broken-81e5f33a24e1 is loaded

**Comment by rainemak** [comment]
I doubt that medium.com issues are related. Tile size needs to be increased to decrease uploads to gpu.

**Comment by rojkov** [comment]
LGTM. Works nice with pref("layers.max-active", 20);

File: src/declarativewebutils.cpp [hunk 1]
```diff
@@ -415,6 +416,20 @@ void DeclarativeWebUtils::handleObserve(const QString message, const QVariant da
     }
 }
 
+void DeclarativeWebUtils::updateTileSize()
```
Comment (rainemak) [inline src/declarativewebutils.cpp]: What should this be called? Not exactly `updateTileSize` maybe something like `setTilingPreferences`

**Comment by rainemak** [comment]
Rebased and updated.

`user_pref("layers.max-active", 20);` preference is needed.

File: src/declarativewebutils.cpp (hunk 1 above)
Comment (rainemak) [inline src/declarativewebutils.cpp]: Actually I think that something like `setRenderingPreferences` would be better.

Then this method could set also:

```
    // Don't force 16bit color depth
    mozContext->setPref(QString("gfx.qt.rgb16.force"), QVariant(false));
    // Use external Qt window for rendering content
    mozContext->setPref(QString("gfx.compositor.external-window"), QVariant(true));
    mozContext->setPref(QString("gfx.compositor.clear-context"), QVariant(false));

    // Progressive painting
    mozContext->setPref(QString("layers.progressive-paint"), QVariant(true));
    mozContext->setPref(QString("layers.low-precision-buffer"), QVariant(true));
```

Comment (rojkov) [inline src/declarativewebutils.cpp]: :+1: 
setRenderingPreferences() would make more sense

**Comment by rainemak** [comment]
Updated.

**Comment by rojkov** [comment]
Tested with a non-debug build. Seems to work with https://medium.com/message/everything-is-broken-81e5f33a24e1

LGTM

**Pull request merged**